.github/workflows/cd.yml

```diff
@@ -13,7 +13,7 @@ jobs:
         with:
           node-version: "16"
 
-      - run: npm install
+      - run: npm ci
```

Comment:
`npm ci` will install the **exact** modules listed in the `package-lock.json` file. So you know that your published version will work exactly the same as it did for you locally. Without a lock file, or using `npm install` (which updates the lock file), you can end up with a sub-dependency at a different version which may cause unforseen issues.

`npm ci` basically solves the "works on my machine" issue. Now it works on all machines.